packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -724,3 +728,181 @@ describeNoCompat("SingleCommit Summaries Tests", (getTestObjectProvider) => {
 		);
 	});
 });
+
+describeNoCompat("IdCompressor Summaries", (getTestObjectProvider) => {
+	let provider: ITestObjectProvider;
+	const config: ITestContainerConfig = {
+		...testContainerConfig,
+		runtimeOptions: { ...testContainerConfig.runtimeOptions, enableRuntimeIdCompressor: true },
+	};
+	beforeEach(() => {
+		provider = getTestObjectProvider();
+	});
+
+	it("Summary includes IdCompressor when enabled", async () => {
+		const { mainContainer } = await createMainContainerAndSummarizer(provider, config);
+
+		const defaultDataStore = await requestFluidObject<ITestDataObject>(
+			mainContainer,
+			defaultDataStoreId,
+		);
+		const containerRuntime = defaultDataStore._context.containerRuntime as ContainerRuntime;
+
+		await provider.ensureSynchronized();
+
+		const { summary } = await containerRuntime.summarize({
+			runGC: false,
```

Comment:
You should not pass in these options. Let summarize use defaults. Not a big deal but its better to test with default unless you actually want to test something else. Same for other tests in this file.

Reply:
Rather than call `summarize` on the runtime, these tests have been rewritten to use the `summarizeNow` helper and to use its result. They've also been moved to `idCompressor.spec.ts`